### PR TITLE
updating title on homepage

### PR DESCRIPTION
**Pull request by catcrooke**
@jxmot @JamBur @cpericak823

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,9 +1,10 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <!-- ----------------------------------------------------------------- -->
-    <title>Just a Title for the Page</title>
+    <title>Music Events Near You</title>
```
Comment (jxmot) [inline index.html]: I like this title.

Comment (cpericak823) [inline index.html]: your title sucks

Comment (jxmot) [inline index.html]: +1

**Comment by jxmot** [comment]
I will merge the change.

**Comment by catcrooke** [comment]
good work CAt

**Pull request merged**